backend/game_server/lib/helpers/vector.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end
```

Comment:
```
defmodule Helpers.NotVector do 
	defexception message: "One of arguments is not Vector" 
end
```

VS

`defmodule Helpers.NotVector do defexception message: "One of arguments is not Vector" end`

I'm not sure but i will do this